omnigibson/envs/env_base.py

```diff
@@ -30,6 +31,8 @@ def __init__(
         automatic_reset=False,
         flatten_action_space=False,
         flatten_obs_space=False,
+        external_camera_kwargs=[],
+        external_camera_poses={},
```

Comment:
maybe make this also a list so we can directly read out at the same time with external camera kwargs?

Reply:
in fact can we just combine them into a single list or dict that has the pose & kwargs for each camera?